### DISCUSSION TITLE
CSV Export

### DISCUSSION THREAD

**Discussion by Nebula-Ninjas** [OpenCSPM/opencspm#163] [Ideas]
OpenCSPM Devs,

I wanted to put in a feature request around the ability to export the results from a custom JSON files import to a CSV file. My thoughts on what would be exported would be the following fields:

Account Number, Region, Check _(CIS, NIST, and etc..)_, Affected Resources _(USERID MFA not enabled, S3 Bucket name made public)_, Severity _(Critical, High, and etc..)_, Service _(IAM, S3, EC2, and etc..)_, Failure Reason _(IAM Passwords should require adequate length, Complexity, Expiration, and reuse by policy)_, Remediation.

Please let me know if this is something that would be a benefit to the tool and what the turnaround time is for new features being implemented, assuming of course the new features are beneficial to the tooling.

Thanks in advance,

William

**Comment by joshlarsen** [comment]
Hi William,

This is planned. Tracking it in #167.
